test/suite/test_tiered14.py

```diff
@@ -0,0 +1,153 @@
+
+    # Run a set of operations, indicated by a string.
+    #  a = add some number of keys
+    #  u = update some number of keys
+    #  c = checkpoint
+    #  r = reopen
+    #  f = flush_tier
+    #  . = check to make sure all expected values are present
+    #
+    # We require a unique test number so we get can generate a different uri from
+    # previous runs.  A different approach is to drop the uri, but then we need to
+    # remove the bucket and cache, which is specific to the storage source extension.
+    def playback(self, testnum, ops):
+        uri = self.uri.format(testnum)
+        self.progress('Running ops: {} using uri {}'.format(ops, uri))
+        ds = TrackedSimpleDataSet(self, uri, self.multiplier,
+                                  config='key_format=' + self.keyfmt)
+        #ds = TrackedComplexDataSet(self, self.uri, self.multiplier,
+        #                           config='key_format=' + self.keyfmt)
+        ds.populate()
+        inserted = 0
+        idx = -1
+
+        # At the end of the set of operations, do a final check ('.')
+        for op in ops + '.':
+            idx += 1
+            try:
+                if op == 'f':
+                    self.progress('flush_tier')
+                    self.session.flush_tier(None)
+                elif op == 'c':
+                    self.progress('checkpoint')
+                    self.session.checkpoint()
+                elif op == 'r':
+                    self.progress('reopen')
+                    self.reopen_conn()
+                elif op == 'a':
+                    self.progress('add')
+                    n = random.randrange(1, 101)  # 1 <= n <= 100
+                    ds.store_range(inserted, n)
+                    inserted += n
+                elif op == 'u':
+                    self.progress('update')
+                    # only update the elements if enough have already been added.
+                    n = random.randrange(1, 101)  # 1 <= n <= 100
+                    if n < inserted:
+                        pos = random.randrange(0, inserted - n)
+                        ds.store_range(pos, n)
+                elif op == '.':
+                    self.progress('check')
+                    ds.check()
+            except Exception as e:
+                self.progress('Failed at position {} in {}: {}'.format(idx, ops, str(e)))
+                raise(e)
+
+    # Test tiered storage with checkpoints and flush_tier calls.
+    def test_tiered(self):
+        random.seed(0)
+
+        # Get started with a fixed set of basic operations.
+        testnum = 0
+        self.playback(testnum, "aaaaacaaa.uucrauaf.aauaac.auu.aacrauafa.uruua.")
+
+        for i in range(0, 10):
+            testnum += 1
+            # Generate a set of 100 operations that is heavy on additions and updates.
+            s = ''.join(random.choices('aaaaauuuuufcr.', k=100))
+            self.playback(testnum, s)
+
+        # FIXME-WT-8758: test disabled for now.
```

Comment:
Where is anything disabled? Please be explicit about what someone needs to change to enable this in the future. It looks like it is enabled to me.

Reply:
Oops, it wasn't disabled in this commit.  Fixed in latest push.